### PR TITLE
Update envoy gateway version

### PR DESCRIPTION
**Pull request by mahmut-Abi**
update envoy version from 0.6.0 to 1.0.0

**What type of PR is this?**

/kind documentation

### CODE REVIEW

File: site-src/implementations.md [hunk 1]
```diff
@@ -231,8 +231,8 @@ few simple steps.
 
 [eg-home]:https://gateway.envoyproxy.io/
 [envoy-org]:https://github.com/envoyproxy
-[eg-supported]: https://gateway.envoyproxy.io/v0.6.0/user/gatewayapi-support
-[eg-quickstart]:https://gateway.envoyproxy.io/v0.6.0/user/quickstart
+[eg-supported]: https://gateway.envoyproxy.io/v1.0.0/api/extension_types
```
Comment (arkodg) [inline site-src/implementations.md:234]: should be https://gateway.envoyproxy.io/v1.0.0/user/traffic/gatewayapi-support/

**Comment by arkodg** [comment]
thanks for updating this doc @mahmut-Abi, can you also update the Envoy Gateway implementation from Beta to GA, tia

**Comment by mahmut-Abi** [comment]
i am bot sure it has been GA.

**Comment by mahmut-Abi** [comment]
i will rebase it

**Review by arkodg — APPROVED** [review]
/approve

Thanks !

**Comment by robscott** [comment]
Congrats to everyone working on Envoy Gateway! @arkodg do you want to submit an updated conformance report along with this?

/ok-to-test
/release-note-none
/approve

**Comment by arkodg** [comment]
@robscott I can raise a follow up for the conformance report, ( nothing has changed from the test coverage perspective )

**Comment by robscott** [comment]
Sounds good, thanks @arkodg!

/lgtm

**Pull request merged**